files/zh-cn/web/javascript/reference/global_objects/bigint/tolocalestring/index.html

```diff
@@ -57,7 +57,7 @@ <h3 id="Using_locales">Using <code>locales</code></h3>
 
 // the nu extension key requests a numbering system, e.g. Chinese decimal
 console.log(bigint.toLocaleString('zh-Hans-CN-u-nu-hanidec'));
-// → 一二三,四五六,七八九,一二三,四五六,七八九
+// → 一二三，四五六，七八九，一二三，四五六，七八九
```

Comment:
这里不应该被更新：

![image](https://user-images.githubusercontent.com/15844309/170846899-8afd871e-4e4a-420e-afbe-bde07ff1c2ef.png)

```suggestion
// → 一二三,四五六,七八九,一二三,四五六,七八九
```

Reply:
这是运行代码的输出结果，其与测试结果一致（It's the output of `bigint.toLocaleString('zh-Hans-CN-u-nu-hanidec')`, we should consider this case）。